components/org.wso2.carbon.siddhi.editor.core/src/main/java/org/wso2/carbon/siddhi/editor/core/util/designview/codegenerator/generators/query/subelements/QuerySubElementCodeGenerator.java

```diff
@@ -104,6 +104,23 @@ public static String generateQueryLimit(long limit) {
         return SiddhiCodeBuilderConstants.EMPTY_STRING;
     }
 
+    /**
+     * Generates the Siddhi code representation of a query's offset
+     *
+     * @param offset The offset value given
+     * @return The Siddhi code representation of the given query's offset value
+     */
+    public static String generateQueryOffset(long offset) {
+        if (offset != 0) {
+            return SiddhiCodeBuilderConstants.NEW_LINE +
```

Comment:
remove new lines

Reply:
Fixed in 8dd3e78e1a28eccb6c7014c24a878fd139399e87